go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/nerdctl
 
-go 1.19
```

Comment:
This is intended to be the minimum supported version

Reply:
Why ? is there sthg blocking on containerd side ?

Reply:
Sorry I miss understood your comment! ok but is it the right place to put  the minimum supported version ? the `go.mod` ?

Reply:
SGTM `A list of minimum versions of other modules required by the current module.` from official [golang doc](https://go.dev/doc/modules/gomod-ref)

Reply:
Thanks for your feedback